pkg/sectigo/server/auth_test.go

```diff
@@ -0,0 +1,61 @@
+package server_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/trisacrypto/directory/pkg/sectigo"
+	"github.com/trisacrypto/directory/pkg/sectigo/server"
+)
+
+func (s *serverTestSuite) TestLogin() {
```

Comment:
I certainly didn't produce huge amounts of test coverage; just enough to ensure that the sectigo client would work with this fake API.